tests/conftest.py

```diff
@@ -0,0 +1,16 @@
+from src.ocrd_utils.config import config
```

Comment:
We might want to use a separate `OcrdEnvConfig` instance, not the default `config` instance to avoid interfering with other tests

Reply:
True. I will add that.